ansible/roles/enroot/tasks/os/redhat-7.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
This version-specific file isn't doing you any good. The `redhat.yml` file will be found by any RedHat family OS.

Reply:
Done.